bin/gyro

```diff
@@ -16,6 +16,8 @@
 
 abort 'error: gyro requires Ruby 2 or higher.' if RUBY_VERSION < '2.0.0'
 
+$LOAD_PATH.unshift File.expand_path('../lib', File.dirname(__FILE__)) if $PROGRAM_NAME == __FILE__
```

Comment:
This line is no longer be useful. If you want to test gyro locally, please use the following lines : 
```bash
gem build gyro.gemspec
gem install gyro-1.0.0.gem
```

Reply:
or you can use an other command when your develop gyro. Go to gyro repo and launch : 
```bash
ruby -Ilib bin/gyro -m <model> -t android -o <output>
```